clients/src/test/java/org/apache/kafka/common/requests/ApiVersionsResponseTest.java

```diff
@@ -112,7 +112,7 @@ public void shouldHaveCommonlyAgreedApiVersionResponseWithControllerOnForwardabl
     }
 
     @ParameterizedTest
-    @EnumSource(names = {"ZK_BROKER", "BROKER"})
+    @EnumSource(names = {"BROKER"})
```

Comment:
How about using `@Test` and replace the `listenerType` by `ListenerType.BROKER` directly?